### PR TITLE
Event visual story link

### PR DESCRIPTION
**Pull request by gestchild**
For #10328
Screenshot 2023-10-18 at 13 52 33

### CODE REVIEW

**Review by rcantin-w — COMMENTED** [review]
Could it be a separate component shared by Event and Exhibition (with the types and styles in it), instead of being exported from Exhibition?

**Review by rcantin-w — APPROVED** [review]
I thought maybe the block itself might be a repeatable component, but it's a bit different isn't in as it only had Visual Story as a possibility?

Looks great!

File: content/webapp/components/styled/AccessResources.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import PlainList from '@weco/common/views/components/styled/PlainList';
+import Space from '@weco/common/views/components/styled/Space';
+import { PaletteColor } from '@weco/common/views/themes/config';
+
+export const ResourcesList = styled(PlainList)`
+  display: flex;
+  flex-wrap: wrap;
+  gap: 15px;
```
Comment (davidpmccormick) [inline content/webapp/components/styled/AccessResources.ts:9]: `gap` can be added with the `Space` component now – is that possible here?

**Review by davidpmccormick — APPROVED** [review]

File: content/webapp/services/prismic/fetch/events.ts [hunk 2]
```diff
@@ -41,7 +43,28 @@ const fetchLinks = [
...
+  event?: EventPrismicDocument;
+  visualStories: prismic.Query<VisualStoryDocument>;
+};
+export async function fetchEvent(
+  client: GetServerSidePropsPrismicClient,
+  id: string
+): Promise<FetchEventResult> {
+  const eventPromise = eventsFetcher.getById(client, id);
+  const visualStoriesQueryPromise = fetchVisualStories(client, {
+    filters: [prismic.filter.at('my.visual-stories.related-document', id)],
```
Comment (rcantin-w) [inline content/webapp/services/prismic/fetch/events.ts:56]:
```suggestion
    filters: [prismic.filter.at('my.visual-stories.relatedDocument', id)],
```

File: content/webapp/components/styled/AccessResources.ts (hunk 1 above)
Comment (gestchild) [reply content/webapp/components/styled/AccessResources.ts:9]: I can't combine PlainList and Space can I?

**Pull request merged**